.eslintrc.cjs

```diff
@@ -1,102 +0,0 @@
-module.exports = {
-  settings: {},
-  env: {
-    node: true,
-  },
-  extends: [
-    "eslint:recommended",
-    "plugin:import/recommended",
-    "plugin:@typescript-eslint/eslint-recommended",
-    "plugin:@typescript-eslint/recommended",
-    "plugin:@typescript-eslint/recommended-requiring-type-checking",
```

Comment:
in the eslint-config-neolution it only uses the recommended but not the recommended-requiring-type-checking, maybe is better to add the recommended-requiring-type-checking in the eslint-config-neolution?

Reply:
added 😄